forms-flow-web/src/components/Application/ApplicationTable.js

```diff
@@ -314,47 +312,19 @@ const ApplicationTable = () => {
         </table>
 
         {applications.length ? (
-          <div className="d-flex justify-content-between align-items-center  flex-column flex-md-row">
-            <div className="d-flex align-items-center">
-              <span className="me-2"> {t("Rows per page")}</span>
-              <Dropdown size="sm">
-                <Dropdown.Toggle variant="light" id="dropdown-basic" data-testid="page-limit-dropdown-toggle">
-                  {pageLimit}
-                </Dropdown.Toggle>
-                <Dropdown.Menu>
-                  {pageOptions.map((option) => (
-                    <Dropdown.Item
-                      key={option.value}
-                      type="button"
-                      onClick={() => {
-                        onSizePerPageChange(option.value);
-                      }}
-                      data-testid={`page-limit-dropdown-item-${option.value}`}
-                    >
-                      {option.text}
-                    </Dropdown.Item>
-                  ))}
-                </Dropdown.Menu>
-              </Dropdown>
-              <span className="ms-2">
-                {t("Showing")} {(limit * pageNo) - (limit - 1)} {t("to")}{" "}
-                {limit * pageNo > totalForms ? totalForms : limit * pageNo}{" "}
-                {t("of")} {totalForms} {t("results")}
-              </span>
-            </div>
-            <div className="d-flex align-items-center">
-              <Pagination
-                activePage={pageNo}
-                itemsCountPerPage={limit}
-                totalItemsCount={totalForms}
-                pageRangeDisplayed={5}
-                itemClass="page-item"
-                linkClass="page-link"
-                onChange={handlePageChange}
-              />
-            </div>
-          </div>
-        ) : null}
+          <table className="table">
+            <tfoot>
+             <TableFooter
+            limit={limit}
```

Comment:
if there is no data need to render some message right @arun-s-aot

Reply:
It will be done by fahad in his pr